### PR TITLE
Patching a missed case in the pythonic solution

### PR DESCRIPTION
**Pull request by ChrisAlertus**
Added a new test case: where there are multiple, different non-alphabet characters in the string. The pythonic solution only accounts for spaces. This PR accounts for any non-alphabet characters.

### CODE REVIEW

File: chapter_01/p04_palindrome_permutation.py [hunk 1]
```diff
@@ -36,7 +36,8 @@ def char_number(c):
 
 def is_palindrome_permutation_pythonic(phrase):
     """function checks if a string is a permutation of a palindrome or not"""
-    counter = Counter(phrase.replace(" ", "").lower())
+    chars = set([chr(97 + i) for i in range(26)])
+    counter = Counter([c.lower() for c in phrase if c.lower() in chars])
```
Comment (brycedrennan) [inline chapter_01/p04_palindrome_permutation.py:40]: lets use `string.ascii_lowercase` instead:
```suggestion
    counter = Counter([c for c in phrase.lower() if c in string.ascii_lowercase])
```

**Review by brycedrennan — COMMENTED** [review]
Thanks for your contribution. I've left one suggestion for an improvement.

File: chapter_01/p04_palindrome_permutation.py (hunk 1 above)
Comment (ChrisAlertus) [reply chapter_01/p04_palindrome_permutation.py:40]: Thank you for that! `string.ascii_lowercase` is very convenient syntax to know :)

**Comment by brycedrennan** [comment]
Sorry I should have mentioned you'll need to `import string`

**Comment by ChrisAlertus** [comment]
That was also on me for being lazy and making the change all in the web UI without testing by running the changed file. I rebased to squash the commit without the import and this time it should work.
Let me know if I missed any more gotcha's!

**Comment by brycedrennan** [comment]
There was a conflict with master so I addressed it and merged your solution here https://github.com/careercup/CtCI-6th-Edition-Python/commit/680c923d1a88adc341ee93e7637625b3d1d1ed70

**Pull request closed without merging**